src/base/static/components/organisms/filter-menu.js

```diff
@@ -162,7 +168,10 @@ class FilterMenu extends React.Component {
                       isSelected={isFilterSelected}
                     >
                       <CategoryLabel isSelected={isFilterSelected}>
-                        {placeForm.label}
+                        {this.props.t(
+                          `filterMenuOption${index}`,
```

Comment:
In most cases I think we'll be able to derive unique i18next keys from supplied information.